app/error_handlers.py

```diff
@@ -30,4 +30,12 @@ def method_not_allowed(e):
 
 @app.errorhandler(500)
 def internal_error(e):
-    return make_response(jsonify(status=500, error='Internal Server Error', message='Well, this is embarrassing...'), status.HTTP_500_INTERNAL_SERVER_ERROR)
+	return make_response(jsonify(status=500, error='Internal Server Error', message='Well, this is embarrassing...'), status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+class InvalidPaymentID(Exception):
+    def __init__(self, message, status_code=None, payload=None):
```

Comment:
also - some of the innards of this class might be overkill. i don't see the payload parameter being used anywhere... what were you planning to pass there?

if this error is raised here, it needs to be caught and handled in updates. or we need to designate an error handler to catch these types of errors